testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -66,21 +67,9 @@ public void testColumnMappingModeNone()
         }
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS}, dataProvider = "columnMappingDataProvider")
```

Comment:
nit: Can we split the commits into two - One would be merging the tests and other would be running the tests for `DELTA_LAKE_OSS` groups.

Reply:
If we split into two commits, the 1st commit will lose `DELTA_LAKE_OSS` coverage for name mapping mode.

Reply:
Makes sense. Can we do the reverse operation - like adding `DELTA_LAKE_OSS` for all mappings and then merge the tests.

Reply:
Updated.